auth0.gemspec

```
@@ -11,9 +11,8 @@ Gem::Specification.new do |s|
   s.summary     = 'Auth0 API Client'
   s.description = 'Ruby toolkit for Auth0 API https://auth0.com.'
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
+  s.files         = `git ls-files -z -- LICENSE README.md lib`.split("\x0")
```

`LICENSE README.md lib` is a discussion point. Please let me know if you want to change.

---

For example, including `EXAMPLES.md` may make sense:

```suggestion
  s.files         = `git ls-files -z -- EXAMPLES.md LICENSE README.md lib`.split("\x0")
```